lib/beaker/host.rb

```diff
@@ -217,7 +217,7 @@ def graceful_restarts?
     def fips_mode?
       case self['platform']
       when /el-7/
-        execute("cat /proc/sys/crypto/fips_enabled") == "1"
+        execute("test -f /proc/sys/crypto/fips_enabled && cat /proc/sys/crypto/fips_enabled") == "1"
```

Comment:
If `fips_enabled` does not exist, `test -f` exits with 1, the same with `cat`. Is this okay?